meter.go

```diff
@@ -2,23 +2,46 @@ package flow
 
 import (
 	"fmt"
+	"math"
 	"sync/atomic"
 	"time"
 )
 
+// IdleRate the rate at which we declare a meter idle (and stop tracking it
+// until it's re-registered).
+//
+// The default ensures that 1 event every ~30s will keep the meter from going
+// idle.
+var IdleRate = 1e-13
+
+// Alpha for EWMA of 1s
+var alpha = 1 - math.Exp(-1.0)
+
 // Snapshot is a rate/total snapshot.
 type Snapshot struct {
 	Rate       float64
 	Total      uint64
 	LastUpdate time.Time
 }
 
+type MeterInterface interface {
+	String() string
+	Mark(count uint64)
+	Snapshot() Snapshot
+	Reset()
```

Comment:
I'd drop all of these from this interface and focus on what the sweeper requires. Different meter implementations will need different interfaces (e.g., different data in their snapshots).

Reply:
@Stebalien Gentle reminder, can you, please, check my last commit and say if it's fine with you?